packages/timeline-state-resolver-types/src/hyperdeck.ts

```diff
@@ -82,8 +82,16 @@ export interface TimelineObjHyperdeckTransport extends TimelineObjHyperdeck {
 		deviceType: DeviceType.HYPERDECK
 		type: TimelineContentTypeHyperdeck.TRANSPORT
 
-		/** The status of the hyperdeck. To start a recording, set to TransportStatus.RECORD */
+		/** The status of the hyperdeck. To start a recording, set to TransportStatus.RECORD. To start playback, set to TransportStatus.PLAY. */
 		status: TransportStatus
+		/** How fast to play the currently-playing clip [-5000 - 5000]. Default 1x speed is 100. 0 is stopped. Negative values are rewind. Values above 100 are fast-forward. */
+		speed: number
```

Comment:
The `speed`, `loop` and `singleClip` should be optional with defaults of `100`, `false` and `true` since in most cases there will be no need to use anything else